src/System.Buffers/src/project.json

```diff
@@ -1,11 +1,16 @@
 {
+  "compilationOptions": {
+    "allowUnsafe": true
+  },
   "dependencies": {
-    "System.Diagnostics.Debug": "4.0.0", 
+    "System.Diagnostics.Debug": "4.0.0",
+    "System.Diagnostics.Tools": "4.0.1-beta-23409",
     "System.Reflection.Primitives": "4.0.0",
     "System.Resources.ResourceManager": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Runtime.Extensions": "4.0.0",
     "System.Runtime.InteropServices": "4.0.0",
+    "System.Slices": "0.1.0-d103015-1",
```

Comment:
The hacky way to do it is to make a folder with the correct name and a new project.json that points to the other folder.